src/containers/ClassifierContainer.jsx

```diff
@@ -28,10 +29,11 @@ class ClassifierContainer extends React.Component {
     this.useRotate90 = this.useRotate90.bind(this);
     this.useResetImage = this.useResetImage.bind(this);
     this.useContrast = this.useContrast.bind(this);
-  
+    this.showMetadata = this.showMetadata.bind(this);
+
     //TEMPORARY
     this.state = {
-      TEST_POPUP: null,
+      POPUP: null,
```

Comment:
Ahh. Ok. Good idea here. I wasn't exactly sure how to proceed if they were temporary, constants, etc. That sounds good though.